examples/beam/src/main/java/org/apache/nemo/examples/beam/GenericSourceSink.java

```diff
@@ -87,6 +93,7 @@ public String apply(final Text text) {
 
   /**
    * Write data.
+   * This method could later be replaced using the HadoopFormatIO class.
```

Comment:
Better to write a TODO comment and file a jira?

Reply:
Done.